docs/sphinx/omero-pyramid/index.rst

```diff
@@ -0,0 +1,22 @@
+Internal OMERO pyramid format v1.0.0
+====================================
+
+For files that contain very large images and are not in a format that supports pyramids, OMERO will generate its own
```

Comment:
This first paragraph is largely unrelated to the format description.  Could it be replaced with a more appropriate introduction to the pyramid format or moved elsewhere?